woocommerce/woocommerce-ios
# Dark Mode: update illustrations for Dark and Light modes

Illustrations for #1555 

## Changes

- Updated illustrations for these two assets:
  - `woo-waiting-customers`: used in Orders, Reviews, Product Loader (from Order Details) empty states
  - `woo-no-store`: used when there are no stores in Store Picker, and Dashboard Top Performers stats empty state

## Testing

- Launch the app
- Go to one of the empty state cases for `woo-waiting-customers` in the section above, or hard code to show the empty state --> the illustration should look good in both Dark and Light modes
- Go to one of the empty state cases for `woo-no-store` in the section above, or hard code to show the empty state --> the illustration should look good in both Dark and Light modes

## Example screenshots

Screen | Light | Dark
-- | -- | --
No orders | ![Simulator Screen Shot - iPhone 8 - 2019-12-11 at 14 44 27](https://user-images.githubusercontent.com/1945542/70599072-45e29180-1c27-11ea-8789-f2753de0515d.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-11 at 14 44 35](https://user-images.githubusercontent.com/1945542/70599073-45e29180-1c27-11ea-9a66-24ca2fab779e.png)
Top performers empty state | ![Simulator Screen Shot - iPhone 8 - 2019-12-11 at 14 53 46](https://user-images.githubusercontent.com/1945542/70599075-467b2800-1c27-11ea-9f94-4117494e2896.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-11 at 14 55 18](https://user-images.githubusercontent.com/1945542/70599077-467b2800-1c27-11ea-8420-a808b5e49470.png)
Empty stores in Store Picker | ![Simulator Screen Shot - iPhone 8 - 2019-12-11 at 14 55 03](https://user-images.githubusercontent.com/1945542/70599076-467b2800-1c27-11ea-871e-d08b5c430b67.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-11 at 14 55 21](https://user-images.githubusercontent.com/1945542/70599078-467b2800-1c27-11ea-8548-a435a7fb3a34.png)







Update release notes:
